JS-Array-Prototype-ForEach.md

```diff
@@ -52,3 +52,18 @@ numbers.forEach(
 document.write(sum);
 // Output: 33
 ```
+
+## How to create your own forEach for learning purposes
+```javascript
+function forEach(array, callback){
+  for(let i = 0; i < array.length; i++){
+    //apply the passed in callback on each element in the array
+    callback(array[i]);
+  }
+}
+function logValue(value, index){
+  console.log(`Element is: ${value}` )
```

Comment:
ES6 string interpolation:`console.log(`Element is: ${value}` )`
Pre-ES6 string interpolation: `console.log("Element is:, value")

When you have more than 1 variable you want included in the log, I believe it makes for sense to do ES6 string interpolation. I also find it easier to read. through.

```js
function logValue(value, index){
  console.log(`Element is: ${value} and the index is: ${index}` )
```
instead of something like:
```js
function logValue(value, index){
  console.log(`Element is: " + value +  "and the index is: " +  index )
```